agent/agent.go

```diff
@@ -1183,6 +1183,7 @@ func (a *Agent) consulConfig() (*consul.Config, error) {
 	base.SerfLANConfig.MemberlistConfig.ProbeTimeout = a.config.GossipLANProbeTimeout
 	base.SerfLANConfig.MemberlistConfig.SuspicionMult = a.config.GossipLANSuspicionMult
 	base.SerfLANConfig.MemberlistConfig.RetransmitMult = a.config.GossipLANRetransmitMult
+	base.SerfLANConfig.MemberlistConfig.MetaMaxSize = a.config.GossipLANMetaMaxSize
```

Comment:
Setting the option for `SerfLANConfig` and `SerfWANConfig` automatically applies it to all the serfs that are being created.